tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -612,20 +612,21 @@ def _write_and_random_read(self, scale: ScaleParameters, topic_names):
             # minutes during these events.
             expect_transmit_time += 600
 
-        seq_consumer = KgoVerifierSeqConsumer(
+        verifier = KgoVerifierConsumerGroupConsumer(
             self.test_context,
             self.redpanda,
             target_topic,
             0,
+            readers=math.ceil(scale.partition_limit / 5000),
```

Comment:
Doesn't this result in 5000 partitions per reader still? I thought that was way too many per the cover letter.

Reply:
It results in 5000 / 6 (broker count) per reader so not entirely 1k+ but still high. As mentioned in the cover letter I still want to keep this around a little bit but just not extreme enough to hit the edge case.